app/project_type/git.py

```diff
@@ -169,24 +235,34 @@ def _execute_git_remote_command(self, command, cwd=None, timeout=10):
         :param timeout: the number of seconds to wait for expected prompts before assuming there will be no prompt
         :type timeout: int
         """
-        child = pexpect.spawn(command, cwd=cwd)
+        # todo: Reenable the functionality around listening for a kill_event to abort execution of this method. This
+        # todo: has been temporarily disabled due to complexity around doing this between multiple processes.
```

Comment:
I like how you separated the regex patterns. It improves the readability of the method.